doc/source/sdk/v2/zaf_v2.md

```diff
@@ -0,0 +1,146 @@
+## Zendesk App Framework v2 Beta
+The Zendesk App Framework (ZAF) v2 introduces a new way of building Zendesk apps using iframes.
+
+The new framework allows you to build powerful apps using whatever technologies you are used to along with all the capabilities you expect from the existing framework.
```

Comment:
Should we enumerate some other advantages of iframe apps, like:

> The v2 Framework improves upon v1 by:
- improving on security by taking advantage of the browsers own cross-domain security features.
- making it easy to use whatever technologies you're familiar with.
- providing consistent APIs across all supported Zendesk Products

?

Alternatively, another section like

> ### How does it improve on v1?

?